lib/jekyll/tags/highlight.rb

```diff
@@ -110,13 +124,14 @@ def render_codehighlighter(code)
 
       def add_code_tag(code)
         code_attributes = [
-          "class=\"language-#{@lang.to_s.tr('+', '-')}\"",
+          "class=\"language-#{@lang.to_s.tr("+", "-")}\"",
```

Comment:
No, it didn't warn, Rubocop only noticed single quotes here.
Sorry, I'm not too deep in Ruby yet, trying to learn it here 😄 
Do you suggest to create this code tag with single interpolation string?
Like
`
%Q{<figure class="highlight"><pre><code class="language-#{@lang.to_s.tr("+", "-")}" data-lang="#{@lang}">#{code.chomp}</code></pre></figure>}
` 
? It would be too long then. And if I change it to multiline string with heredoc then a lot of tests are failing because they expect a single line string.
Should we change it to multiline here and update the tests?